README.md

```diff
@@ -0,0 +1,9 @@
+# Acorn for Moodle
+======
+
+Moodle™ LMS is an open source online Learning Management System widely used at universities, schools, and corporations. It is modular and highly adaptable to any type of online learning.
+
+## Configure the Moodle
+
+Moodle uses mariadb as the database and it is been used as the [services](https://docs.acorn.io/reference/acornfile#services-consuming) . If you are using Advanace Options on Acorn Platform you can provide the `moodledbname`. By default it will be named as `moodledb`.
+By default username and password for moodle is `username: user` `password: bitnami`. You can changes it using `moodle_username` and `moodle_password` field.
```

Comment:
In addition to these, let's also enable configuring [User & Site Configs](https://github.com/bitnami/containers/tree/main/bitnami/moodle#user-and-site-configuration) as well as the [mysql db configs](https://github.com/bitnami/containers/tree/main/bitnami/moodle#create-a-database-for-moodle-using-mysql-client). Of course, not all of the config options in the docker image would make sense for Acornfile. Use your judgement.

Reply:
Updated User and Site Config. For DB as we  are using it as  a services so already provided all the field that is mentioned in [Using an Existing Database ](https://github.com/bitnami/containers/tree/main/bitnami/moodle#use-an-existing-database) Section.